src/layouts/components/Homepage/HeroBody.tsx

```diff
@@ -37,6 +42,276 @@ export interface HeroBodyFormFields {
+      <FormContext
+        hasError={!!errors.background}
+        onFieldChange={onChange}
+        isRequired
+      >
+        <Box mb="0.5rem">
+          <FormTitle>Hero background image</FormTitle>
+        </Box>
+        <FormFieldMedia
+          value={background}
+          id={`section-${index}-hero-background`}
+          inlineButtonText="Select"
+        />
+        <FormError>{errors.background}</FormError>
+      </FormContext>
+    </Box>
+  )
+}
+
+type SectionSize = "half" | "one-third"
+
+type SectionAlignment = "left" | "right"
+
+type SectionBackgroundColor = "black" | "white" | "translucent gray"
+
+interface HeroSideSectionProps extends HeroCenteredLayoutProps {
+  background: string
+  index: number
+  errors: {
+    title: string
+    subtitle: string
+    background: string
+  } & HeroBodyFormFields
+  size: SectionSize
+  alignment: SectionAlignment
+  backgroundColor: SectionBackgroundColor
+}
+
+const HeroSideSectionLayout = ({
+  background,
+  index,
+  errors,
+  title,
+  subtitle,
+  size = "half",
+  alignment = "left",
+  backgroundColor = "black",
+}: HeroSideSectionProps) => {
+  const [, setSectionSize] = useState(size)
+  const [, setSectionAlignment] = useState(alignment)
+  const [, setSectionBackgroundColor] = useState(backgroundColor)
+
+  return (
+    <>
+      <HeroCenteredLayout
+        index={index}
+        background={background}
+        errors={errors}
+        title={title}
+        subtitle={subtitle}
+      />
+      <Box>
+        <Text textStyle="subhead-1">Section size</Text>
+        <Radio.RadioGroup
+          onChange={(nextSectionSize) => {
+            setSectionSize(nextSectionSize as SectionSize)
+          }}
+          defaultValue="half"
+        >
+          <HStack spacing="0.5rem">
+            <Radio value="half" size="xs" w="50%" allowDeselect={false}>
+              Half (1/2) of banner
+            </Radio>
+            <Spacer />
+            <Radio value="one-third" size="xs" w="50%" allowDeselect={false}>
+              Third (1/3) of banner
+            </Radio>
+          </HStack>
+        </Radio.RadioGroup>
+      </Box>
+      <Box w="100%">
+        <Text textStyle="subhead-1">Alignment</Text>
+        <Radio.RadioGroup
+          onChange={(nextSectionAlignment) => {
+            setSectionAlignment(nextSectionAlignment as SectionAlignment)
+          }}
+          defaultValue="left"
+        >
+          <HStack spacing="0.5rem">
+            <Radio value="left" size="xs" w="50%" allowDeselect={false}>
+              Left
+            </Radio>
+            <Spacer />
+            <Radio value="right" size="xs" w="50%" allowDeselect={false}>
+              Right
+            </Radio>
+          </HStack>
+        </Radio.RadioGroup>
+      </Box>
+      <Box w="100%">
+        <Text textStyle="subhead-1">Section background colour</Text>
+        <HStack spacing="0.75rem" alignItems="flex-start">
+          <IconButton
```

Comment:
maybe can store as [{}] and .map

Reply:
ps, could you go into a bit more detail her?

Reply:
i will put as a function cos it relies on the icon colour (black/gray/white)